SCSS Build Error

First reported here: https://twitter.com/jesephm/status/1263190672317853700

Additional issue added here: #528

In my own testing, I found that (sometimes) there is a set of errors triggered by running `npm i`.

```
Input Error: You must pass a valid list of files to parse
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wd_s@1.0.0 optimizecss: `postcss dist/css/style.css -c postcss.config.js --map=true -o dist/css/style.css`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the wd_s@1.0.0 optimizecss script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/coreycollins/.npm/_logs/2020-05-21T16_21_45_318Z-debug.log
ERROR: "optimizecss" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wd_s@1.0.0 build:scss: `run-s scss gutenberg optimizecss`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the wd_s@1.0.0 build:scss script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/coreycollins/.npm/_logs/2020-05-21T16_21_45_350Z-debug.log
ERROR: "build:scss" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wd_s@1.0.0 build: `run-p build:scss build:js build:images build:pot frontend clean`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the wd_s@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/coreycollins/.npm/_logs/2020-05-21T16_21_45_447Z-debug.log
ERROR: "build" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wd_s@1.0.0 postinstall: `run-s frontend build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the wd_s@1.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/coreycollins/.npm/_logs/2020-05-21T16_21_45_488Z-debug.log
```

I was able to replicate this *sometimes*. It seems as though it was only happening for me in my `Desktop` directory, but any other directory on my system worked fine. In the instance where it didn't work (on desktop), running `sudo npm i` got around the issue.

For some reason, everything in `/dist/` was being compiled except for `style.css`.

In testing on my local, removing the recently added source maps addition to our `scss` script resolves this issue. Now, on my desktop, I'm able to run `npm i` without any issues.

--- a/package.json
+++ b/package.json
@@ -70,7 +70,7 @@
     "polyfill": "uglifyjs src/js/babel-polyfill.min.js -m -o dist/js/babel-polyfill.js && uglifyjs src/js/babel-polyfill.min.js -m -c -o dist/js/babel-polyfill.min.js",
     "postinstall": "run-s frontend build",
     "pot": "mkdirp dist/languages && wp-pot --src './*.php' --dest-file 'dist/languages/_s.pot'",
-    "scss": "node-sass src/scss/style.scss -o dist/css --source-map-root file://${PWD}/ --source-map-embed true",
+    "scss": "node-sass src/scss/style.scss -o dist/css",
     "serve": "browser-sync start --https --proxy 'https://_s.test' --no-open --files \"dist/css/*.css, dist/js/*.js, **/*.html, **/*.php, !node_modules/**/*.html\"",
     "uglify": "mkdirp dist/js -p && uglifyjs src/js/concat/*.js -m -o dist/js/app.js && uglifyjs src/js/concat/*.js -m -c -o dist/js/app.min.js",
     "watch": "run-p serve watch:*",

